README.md

```diff
@@ -20,16 +36,16 @@ pip install lxml --user
 pip install sickle --user
```

Comment:
Would recommend using virtualenv and specifying those requirements in `requirements.txt`. (I can help with this.)